app/controllers/photos_controller.rb

```diff
@@ -24,7 +24,10 @@ def edit
   def create
     @photo = Photo.new(photo_params)
     @photo.owner = current_user
-
+    
+    authorize @photo
+    skip_authorization 
```

Comment:
Why skip authorization?

Reply:
<img width="537" alt="Screenshot 2024-03-08 at 12 11 27 AM" src="https://github.com/andrewp8/industrial-auth-1/assets/69804999/c1facf29-f856-4202-bdb3-8012c5964428">
I encountered an issue while creating a new record, and that was my approach at the time. I now realize this approach goes against Pundit ideology, and everything became clear after your explanation yesterday.